cmd/mount.go

```diff
@@ -400,7 +403,27 @@ var mountCmd = &cobra.Command{
 			}
 
 			ctx, _ := context.WithCancel(context.Background()) //nolint
-			daemon.SetSigHandler(sigusrHandler(pipeline, ctx), syscall.SIGUSR1, syscall.SIGUSR2)
+			var sigusr2, sigchild chan os.Signal
+			if !daemon.WasReborn() { // execute in parent only
+				// redirect libfuse stderr to a temp file
+				f, err := ioutil.TempFile("", "libfuse_stderr_")
+				if err != nil {
+					log.Err("failed to create temp file for storing libfuse error log [%v]", err)
+				} else {
+					defer os.Remove(f.Name())
+					dmnCtx.LogFileName = f.Name()
+				}
```

Comment:
This code is resulting into a panic in my env. Let me create a branch out of this and see what is causing it.

Reply:
I just used PIPE instead of temp file for communication between child and parent, so no more temp file, please take another look.

Reply:
I guess temp file shall be fine, PIPE will have this issue of one side getting blocked if another side goes away in between. Leave to you on what might be better approach. I am more concerned about the panic that I hit in my testing.

Reply:
Thanks, I've tested that it won't be blocked from parent side once child exited abnormally, child will always put something to stderr, which is the writing side of pipe. Also, parent will always exit when timeout. So, I think pipe won't be a problem.

Reply:
> I am more concerned about the panic that I hit in my testing. 

You could put the panic log here if you want and we can discuss together.

Reply:
Pipe can probably solve the panic problem, since no more temp file creation and deletion.